MagicTrialGame/Models/Data/GameData.cs

```diff
@@ -0,0 +1,21 @@
+using MagicTrialGame.Models.Rooms;
+
+namespace MagicTrialGame.Models
+{
+    public class GameData
+    {
+        public Player Player { get; private set; }
```

Comment:
Nit ⛏️ - pri vytvoreni `GameData` je player `null` a pak spolehame na to, ze nam ho nekdo doda predtim, nez ho pouzije.